### ISSUE TITLE
Incorrectly has `react-dom` as a peer dependency

### ISSUE THREAD

**Issue by donavon** [imbhargav5/rooks#120]
**Describe the bug**
`react-dom` is NOT a peer dependency for `useCounter`. With this in place, it shows a the following warning in the console:

```
npm WARN @rooks/use-counter@3.2.4 requires a peer of react-dom@>=16.8.0 but none is installed. You must install peer dependencies yourself.
```

This means that you get this warning when using `useCounter` in React Native and Ink (CLI).

**Comment by imbhargav5** [comment]
Absolutely. Would you like to submit a PR?

**Comment by imbhargav5** [comment]
@donavon A fix will deployed in the next release. 🎉

**Comment by imbhargav5** [comment]
Fixed with e0cdf32f97e625996d701e14f5d7c104c026bb96
